src/templates/project.js

```diff
@@ -7,7 +7,14 @@ export default ({ data }) => {
       <h1>
         {project.frontmatter.title}
       </h1>
-      <div dangerouslySetInnerHTML={{ __html: project.html }} />
+      {/*
+          TODO: Need to figure out a way to translate a frontmatter field from
+          markdown to  HTML the way it does automatiically when the markdown is
+          beneath the frontmatter. There are cases where we might need more than
+          one markdown field. Otherwise, we have to look into how Netlify CMS
+          creates md files.
+      */}
```

Comment:
For project .md files that have text beneath the frontmatter, the gatsby-remark plugin nicely add the html to the GraphQL node. But now, I need to solve how the transform frontmatter values to html